### PR TITLE
cluster: add metric for current network

### PR DESCRIPTION
**Pull request by xenowits**
Add metric for current `network`/`chain`. Also, add updated simnet dashboard.

category: feature
ticket: none

### CODE REVIEW

**Comment by xenowits** [comment]
Screenshot 2022-09-24 at 3 38 39 PM

File: dkg/dkg.go [hunk 1]
```diff
@@ -609,7 +609,7 @@ func dvsFromShares(shares []share) ([]cluster.DistValidator, error) {
 	return dvs, nil
 }
 
-func forkVersionToNetwork(forkVersion []byte) (string, error) {
```
Comment (xenowits) [inline dkg/dkg.go:612]: should we move `forkVersionToNetwork` and `networkToForkVersion` functions to a "utils" file?

File: app/metrics.go [hunk 2]
```diff
@@ -77,17 +78,42 @@ var (
...
+}
+
+// setNetworkGauge sets the gauge to the current network/chain (ex: goerli, mainnet etc.).
+func setNetworkGauge(forkVersion []byte) error {
+	network, err := dkg.ForkVersionToNetwork(forkVersion)
+	if err != nil {
+		return err
+	}
+
+	networkGauge.WithLabelValues(network)
```
Comment (corverroos) [inline app/metrics.go:116]: no need to error, this is just metadata, rather set to 'unknown'.

**Comment by corverroos** [comment]
Not a fan of that dashboard layout, the log graphs are too tall and narrow. Also suggest `# Operators` and `# Validators`

**Review by corverroos — APPROVED** [review]

**Comment by xenowits** [comment]
Screenshot 2022-09-24 at 9 13 55 PM

**Comment by xenowits** [comment]
Screenshot 2022-09-24 at 9 17 57 PM

**Pull request merged**